### ISSUE TITLE
MailChimp Connection: Are we displaying testing screenshots for production environment? 

### ISSUE THREAD

**Issue by htdat** [Automattic/wp-calypso#35402]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/marketing/connections/domain.com
2. Look at Mailchimp, there is a screenshot with Campaigns, Templates, etc and "Dave Test. Co".
3. Screenshot is here https://wordpress.com/calypso/images/sharing/mailchimp-screenshot.png

#### What I expected

Not sure what is expected here. 

#### What happened instead

But at least we should not display a testing screenshot here. 

#### Screenshot / Video

This is what I am seeing in a normal Jetpack Pro site without any proxy. 

![Screen Shot 2019-08-15 at 09 38 15](https://user-images.githubusercontent.com/10045087/63069424-fa51fc80-bf40-11e9-8e6e-4e076eb32846.png)

**Comment by htdat** [comment]
Internal notification: p1565837071157800-slack-calypso

**Comment by simison** [comment]
Cc @davemart-in

**Comment by arunsathiya** [comment]
The screenshot looks fine to me. `Dave Test Co.` sounds like a generic company name, similar to `Acme Commerce`. 

Are we expecting something else here?

**Comment by bluefuton** [comment]
As @arunsathiya says, I think this is just demonstrating the functionality of the Mailchimp plugin. Is there a way we can make it clearer that it's an example?

**Comment by davemart-in** [comment]
> I think this is just demonstrating the functionality of the Mailchimp plugin

Yep, that was the goal. This screenshot could be removed, or we could change "Dave Test Co." to something more generic like "Acme Inc" as @arunsathiya suggested.

cc @artpi

**Comment by htdat** [comment]
> The screenshot looks fine to me. Dave Test Co. sounds like a generic company name, similar to Acme Commerce.

It's not _fine_ to me. In the first time, I tried to click everything in the screenshot and nothing worked then I tried to figure out what it is. Finally, it was a screenshot. 

In my opinion, we may put a link to our guide rather than a screenshot like this.

**Comment by arunsathiya** [comment]
This is still a valid issue. Removing stale tag.

**Comment by jordesign** [comment]
Flagging this for Manage focus for review (and possibly janitorial)

**Comment by obenland** [comment]
@sixhours I assume this pre-dates your work on connection screenshots? Do you think you'd have time to change this to something more generic?

/cc @lcollette
